Package.swift

```diff
@@ -23,7 +23,7 @@ let package = Package(
         .package(url: "https://github.com/duckduckgo/TrackerRadarKit", .exact("1.1.1")),
         .package(name: "Punycode", url: "https://github.com/gumob/PunycodeSwift.git", .exact("2.1.0")),
         .package(url: "https://github.com/duckduckgo/content-scope-scripts", .exact("3.2.0")),
-        .package(url: "https://github.com/duckduckgo/privacy-dashboard", .branch("main"))
+        .package(url: "https://github.com/duckduckgo/privacy-dashboard", .branch("shane/esm"))
```

Comment:
Shall that be merged first so we still point to main?

Reply:
Precisely. This branch is where Shane applies all recent design updates. Finally for the release we will schedule for the exact version number (PD will use versioned releases).